docs/user-guide/getting-started.md

```diff
@@ -11,175 +11,173 @@ The Ambassador Edge Stack is designed to run in Kubernetes for production. The m
+    -> Provisioning a cloud load balancer. (This may take a minute, depending on your cloud provider.)
+    -> Automatically configuring TLS.
+    Please enter an email address. We’ll use this email address to notify you prior to domain and certification expiration [None]: john@example.com.
+    ```
 
-  For more information, see [Edge Control](../../reference/edge-control).
+    Minikube users will see something similar to the following:
 
-5. To access the Edge Policy Console going forward, you can access it using one of the following options:
-* `edgectl login -n <namespace> <AES_host>` or
-* `https://{{AES_URL}}/edge_stack/admin`
+    ```bash
+    $ edgectl install
+    -> Installing the Ambassador Edge Stack 1.0.
+    -> Automatically configuring TLS.
+    -> Cluster is not publicly accessible. Please ensure your cluster is publicly accessible if you would like to use automatic TLS.
 
-## Configure TLS Termination and Automatic HTTPS
+    Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit http://192.168.64.2:31334 to access your Edge Stack installation and for additional configuration.
+    ```
 
-**The Ambassador Edge Stack enables TLS termination by default using a self-signed certificate. See the [Host CRD](/reference/host-crd) for more information about disabling TLS.** If you have the ability to update your DNS, Ambassador can automatically configure a valid TLS certificate for you, eliminating the TLS warning. If you do not have the ability to update your DNS, skip to the next section, "Create a Mapping."
+6. Provide an email address as required by the ACME TLS certificate provider, Let's Encrypt. Then your terminal will print something similar to the following:
 
-1. Update your DNS so that your domain points to the IP address for your cluster. 
+    ```shell
+    -> Obtaining a TLS certificate from Let’s Encrypt.
 
-2. In the Edge Policy Console, create a `Host` resource:
-   * On the "Hosts" tab, click the **Add** button on the right.
-   * Enter your hostname (domain) in the hostname field.
-   * Check the "Use ACME to manage TLS" box.
-   * Review the Terms of Service and check the box that you agree to the Terms of Service.
-   * Enter the email address to be associated with your TLS certificate.
-   * Click the **Save** button.
-  
-  You'll see the newly created `Host` resource appear in the UI with a status of "Pending." This will change to "Ready" once the certificate is fully provisioned. If you receive an error that your hostname does not qualify for ACME management, you can still configure TLS following [these instructions](../../reference/core/tls) or by reviewing configuration in the [Host CRD](/reference/host-crd).
+    Congratulations, you’ve successfully installed the Ambassador Edge Stack in your Kubernetes cluster. Visit https://random-word-3421.edgestack.me to access your Edge Stack installation and for additional configuration.
```

Comment:
I think we need to explain what edgestack.me is doing, and explain what `random-word-3421` means.

Reply:
fixed